index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <nav class="header-logo">
+        <img src="images/My Logo.svg" alt="" class="logo">
+        <img src="images/Enabled.png" alt="" class="image-icon"></nav>
+         
+    </nav>
```

Comment:
- [ ] Good job 😃 , but as the previous reviewer said, please make your `logo` a link item with an anchor `a` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.